tensornetwork/network.py

```diff
@@ -840,10 +847,11 @@ def split_node(
     """Split a network_components.Node using Singular Value Decomposition.
 
     Let M be the matrix created by flattening left_edges and right_edges into
-    2 axes. Let U S V* = M be the Singular Value Decomposition of M.
-    This will split the network into 2 nodes. The left node's tensor will be
-    U * sqrt(S) and the right node's tensor will be sqrt(S) * (V*) where V* is
-    the adjoint of V.
+    2 axes. Let :math:`U S V^* = M` be the Singular Value Decomposition of 
+    :math:`M`. This will split the network into 2 nodes. The left node's 
+    tensor will be :math:`U * \\sqrt{S}` and the right node's tensor will be 
```

Comment:
I don't think we need the * between U and sqrt{S} (and likewise between sqrt{S} and V^*). Similarly to how we don't have them between U, S, and V^* in the specification of the SVD above. Matrix multiplication is understood.

Reply:
Done